contrib/search_api_views/search_api_views.info

```diff
@@ -1,9 +1,7 @@
 name = Search Views
 description = Integrates the Search API with Views, enabling users to create views with searches as filters or arguments.
 dependencies[] = search_api:search_api
-dependencies[] = entity_plus:entity_plus
+dependencies[] = entity_plus (>1.x-1.0.10)
```

Comment:
Haven't tested but should this be `>=`?

Reply:
I used this page as a source:
https://api.backdropcms.org/modules

The dependencies[] property in the .info file can also optionally specify versions:

dependencies[] = exampleapi (>1.0)
The above module requires any version greater than version 1.0.

You can optionally specify the core version number as well:
dependencies[] = exampleapi (>1.x-1.5.0)

The above module requires a 1.x version compatible version of the module and a version greater than 1.5.

Additionally, multiple version dependencies can be specified as comma-separated values within the parentheses:

dependencies[] = exampleapi (>1.0, <=3.2, !=3.0)

Reply:
Oh duh I see

Reply:
This has been done